pkg/avx/collector.go

```diff
@@ -253,3 +254,13 @@ func (c collector) collectLastCPUStats(ch chan<- prometheus.Metric) {
 			fmt.Sprintf("CPU%d", binary.LittleEndian.Uint32(lastCPU)))
 	}
 }
+
+func init() {
+	// TODO: collector config
+	c, err := newCollector("/sys/fs/cgroup/unified", "./libexec/avx512.o")
```

Comment:
I guess adding configureability to `init()` may lead to problems in unit tests later. Can we have this collector registration more explicit in the consumer code (in `cmd/gatherer/gatherer.go`)?

Reply:
OK with the change I'm proposed?

Reply:
Yes, looks good now